src/lib/atoms/Button/Button.test.tsx

```diff
@@ -1,50 +1,70 @@
-import React from 'react';
+import React, { MouseEvent } from 'react';
 import { ReactWrapper, mount } from 'enzyme';
 
 // Components
 import Button, { IButtonProps } from './index';
+import GeneUIProvider from '../../providers/GeneUIProvider';
 
-describe('Button', () => {
+describe('Button ', () => {
```

Comment:
I can't find the `isIconAfter` prop test

Reply:
Test with icons i  will add after finish Icon component.